interfaces/matlab_experimental/Utility/callct2.m

```diff
@@ -0,0 +1,24 @@
+function output = callct2(varargin)
```

Comment:
Rather than `callct2` it may make sense to indicate that this returns a string in the function name: `ctstring`?